--- a/src/ObservableQueue.h	2014-10-07 10:20:39.580952874 +0200
+++ b/src/ObservableQueue.h	2014-10-07 10:36:48.751804595 +0200
@@ -331,14 +331,14 @@ CObservableQueue<ValueType>::~CObservabl
 template <typename ValueType>
 void CObservableQueue<ValueType>::ObserverAdded( ObserverType* o )
 {
-	NotifyObservers( EventType( EventType::STARTING ), o );
+	this->NotifyObservers( EventType( EventType::STARTING ), o );
 }
 
 
 template <typename ValueType>
 void CObservableQueue<ValueType>::ObserverRemoved( ObserverType* o )
 {
-	NotifyObservers( EventType( EventType::STOPPING ), o );
+	this->NotifyObservers( EventType( EventType::STOPPING ), o );
 }
 
 
